View/View.php

```diff
@@ -24,14 +24,54 @@
  */
 class View
 {
+    /**
+     * @var mixed|null
+     */
     private $data;
+
+    /**
+     * @var int|null
+     */
+    private $statusCode;
+
+    /**
+     * @var mixed|null
```

Comment:
https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1299/files#diff-5959da22e28b61bcb047ba680731fdcdR380
as you can see, that returns type hint matches the hint of this property.